packages/xarc-app-dev/src/config/get-dev-proxy.ts

```diff
@@ -109,7 +109,8 @@ module.exports = function createDevProxy() {
     webpackDevHost,
     protocol,
     elevated,
-    useDevProxy
+    useDevProxy,
+    devAdminPort: parseInt(process.env.ELECTRODE_ADMIN_PORT || "8991")
```

Comment:
I much prefer `Number(x)` to parseInt(x, 10)

It's uniform to `String(x)` and `Boolean(x)` and `Symbol(x)`